internal/controllers/core/tiltfile/reconciler.go

```diff
@@ -493,12 +493,17 @@ type runStatus struct {
 }
 
 func (rs *runStatus) TiltfileStatus() v1alpha1.TiltfileStatus {
+	var hashes *v1alpha1.TiltfileHashes
+	if rs.tlr != nil && rs.tlr.Hashes.AllFilesSHA256 != "" {
```

Comment:
is there a reason to publish this to the apiserver? seems like it's internal state to the Tiltfile reconciler, and not something other reconcilers need to know about.

i would really like to avoid passing TiltfileStatus to the tiltfile loader at all

Reply:
No, there isn't. I was looking for a clean way to pass previous state into `TiltfileLoader.Load(...)`. I can modify the signature there and store the state in the reconciler runStatus instead with an internal-only struct, would that work for you?

Reply:
ya, i'd be fine with passing some sort of `lastTiltfileLoadResult` struct or `lastLoadInfo` struct

Reply:
Or maybe just the previous TiltfileLoadResult. I think that would work well.